rpc/grpc/client/src/lib.rs

```diff
@@ -81,6 +87,7 @@ impl GrpcClient {
     pub async fn connect(
```

Comment:
Missing some documentation of the arguments for this method. Most devs building this object will be unfamiliar with internals. Perhaps also a simplified `connect_with_defaults` where we select the most basic config would be in place (for instance the subscription context should be `None` )

Reply:
done